### DISCUSSION TITLE
General Discussion

### DISCUSSION THREAD

**Discussion by JohnMertz** [MailCleaner/MailCleaner-Next#26] [General]
This area should serve as a nice place to introduce yourself to the community and have some 💬 General Discussion, related to and unrelated to MailCleaner.

If you don't know where to start, feel free to introduce yourself and let us know your use-case for MailCleaner, your experience with previous versions of MailCleaner, and what interests you are better able to pursue by having less spam to look at every day!

**Comment by JohnMertz** [comment]
To break the ice... 🧊⛏️

I'm John! The Head of Development for MailCleaner! I have been with the company since 2018, which was my first experience with MailCleaner. 

Right out of the gate I got started by extracting the filesystem from the QCOW2 image in order to run a very tiny installation for a handful of personal domains which receive ~100-200 messages/day. This is still not an officially supported approach, but one that I hope we can pursue as time goes on (although it is more likely to be with regular OSI runtime, like Docker/Podman).

Obviously, I now help to administer the MailCleaner Cloud installation which processes millions of messages/day across 6 nodes, and provide support to a wide variety of machines using MailCleaner Enterprise.

When not working on MailCleaner, I busy myself my playing around with all kinds of open source, self-hosted tech. I have several other side projects at varying degrees of completion and varying levels of visibility, but seek to do a lot more development in the open as time goes on. I'm also always busy with real-life, physical projects including electronics, home renos, furniture building, and much more.

**Comment by jloms** [comment]
Hello,
I 'm using mailcleamner CE for many years ... everything quite fine except the dependency to old, really old debian version.
So really happy a bookworm version on production.
Then I've tried the alpha, but it's really hard to get it working without more doc.
Would it be possible to have more detailed doc on architecture and dependencies of this new version ?
Thanks

**Reply by itex-it** [reply]
Hello, I also tried to setup a test appliance but after some hours digging around I stopped.
Looking here regularly but it seems that nothing happens. I´m so sorry that a good product won´t continue
Regards

**Reply by JohnMertz** [reply]
Hello folks,

I understand the frustration. The simple fact of the matter is that we have fewer development resources than we need. I'm essentially the only one working on this, but also need to spend my time working on the current MailCleaner release in order to keep all of the existing users happy and also need to do behind-the-scenes work on infrastructure and other tasks. Progress essentially stopped over the holidays and I still have a couple of other tasks with the current release before I can dedicate myself to the next release again.

As noted in the docs, the Alpha is not really expected to "just work". Community participation at this stage is to help point out additional obvious problems that we may not notice or which may not impact us during our testing. It also allows for interested community members to patch problems that they find on our behalf. We welcome either kind of contribution. More significant contribution would probably help in either case, but if it is documentation regarding problems that are expected to be fixed in the near/medium term, then it does not necessarily make sense for me to spend the time creating those documents just to delete them later. If testers do have some success setting things up, I would welcome them to contribute their methods to the documentation themselves.

Work will resume on this release in the coming weeks and hopefully I can resolve any outstanding critical issues before we start releasing Beta images. Those images should have a more streamlined installation and all components should be expected to at least work (with bugs to be expected).

**Reply by jlo-lirmm** [reply]
Thanks for the answer. I'll wait for beta to test et report problems.
